backend/Makefile

```diff
@@ -82,4 +83,4 @@ docker-clean:
 
 docker-lint:
 	docker compose build
-	docker-compose run web bash -c 'flake8 && black --check . && bandit -c pyproject.toml -r . && mypy . && djlint .'
\ No newline at end of file
+	docker-compose run web bash -c 'flake8 && black --check . && bandit -c pyproject.toml -r . && mypy . && djlint .'
```

Comment:
I think/hope this was picked up elsewhere... in fact, I'm confident it will be on merge?

`docker-compose` should become `docker compose` everywhere. If a merge from `main` doesn't pick this up, then we'll go looking for whether it is lingering in a branch...

Reply:
This branch is [46 commits ahead of](https://github.com/GSA-TTS/FAC/compare/main...tt/1146), [111 commits behind](https://github.com/GSA-TTS/FAC/compare/tt/1146...main) main.

```
git checkout main
git pull
git checkout tt/1146
git merge main
```